dist/ng-token-auth.js

```diff
@@ -331,7 +332,7 @@ angular.module('ng-token-auth', ['ipCookie']).provider('$auth', function() {
               }
               authUrl = this.getConfig(opts.config).apiUrl;
               authUrl += this.getConfig(opts.config).authProviderPaths[provider];
-              authUrl += '?auth_origin_url=' + encodeURIComponent($window.location.href);
+              authUrl += '?auth_origin_url=' + encodeURIComponent(opts.auth_origin_url || $window.location.href);
```

Comment:
Apple does not support `localhost` domains for redirect as part of their OAuth implementation. This has practical implications when doing things in WKWebView proxied environments, where internal URLs are all `localhost`-based. This `opts` flag provides an escape hatch for a scenario such as this.